engines/config-query-sparql/config/config-rdfjs.json

```diff
@@ -30,39 +29,30 @@
     "ccqs:config/query-parse/mediators.json",
     "ccqs:config/query-process/actors.json",
     "ccqs:config/query-process/mediators.json",
-    "ccqs:config/query-result-serialize/actors.json",
+    "ccqs:config/query-result-serialize/actors-rdfjs.json",
     "ccqs:config/query-result-serialize/mediators.json",
     "ccqs:config/query-source-identify/actors-rdfjs.json",
     "ccqs:config/query-source-identify/mediators.json",
     "ccqs:config/query-source-identify-hypermedia/actors-sparql.json",
     "ccqs:config/query-source-identify-hypermedia/mediators.json",
-
```

Comment:
Please keep the empty lines.
They are here to indicate where the differences are to the default config.